src/components/card/arrow.svg

```diff
@@ -0,0 +1,17 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g filter="url(#filter0_d)">
+<circle cx="24" cy="20" r="20" fill="#F3F3F4"/>
```

Comment:
you can create a folder with assets in which you will store all the pictures

Reply:
corrected